contracts/TwabRewards.sol

```diff
@@ -335,10 +351,12 @@ contract TwabRewards is ITwabRewards {
         @return Amount of tokens left to be rewarded
      */
     function _getRemainingRewards(Promotion memory _promotion) internal view returns (uint256) {
-        // _tokensPerEpoch * _numberOfEpochsLeft
-        return
-            _promotion.tokensPerEpoch *
-            (_promotion.numberOfEpochs - _getCurrentEpochId(_promotion));
+        unchecked {
```

Comment:
_getCurrentEpochId can return an epoch **greater than** the number of epochs, so this code needs to take that into account.

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/41/commits/91f880dea8029554e88986c84e77beda9326f888